helm/values.yaml

```diff
@@ -118,7 +118,7 @@ signaleringen:
   restartPolicy: Never
   image:
     repository: curlimages/curl
-    tag: latest
+    tag: latest@sha256:c1fe1679c34d9784c1b0d1e5f62ac0a79fca01fb6377cdd33e90473c6f9f9a69
```

Comment:
No `latest` tag! I think there is another PR already for this maybe because I already made this remark somewhere else I think?

Reply:
Or is this PR not based on the main branch?

Reply:
Or did I already review this PR? I am confused. ;-)

Reply:
Strange, I was sure it was updated with a change request by @niorg ... I'll have another look 👀

Reply:
Manually updated ✔️